### PR TITLE
[TEVA-3107] update application form copy

### PR DESCRIPTION
**Pull request by alexbowen**
https://dfedigital.atlassian.net/browse/TEVA-3107

### CODE REVIEW

**Comment by bencmitchell** [comment]
The copy on Step 6 - References needs to be updated to _”If you are shortlisted, your referees may be contacted before your interview.”"_
image

File: config/locales/jobseekers.yml [hunk 1]
```diff
@@ -133,8 +130,7 @@ en:
         references:
           description1_html: >-
             <span class='text-red'>Two referees are required</span> for this application. One of these must be your
```
Comment (david-mears-dfe) [inline config/locales/jobseekers.yml:132]: Should this span be closed?

**Review by david-mears-dfe — APPROVED** [review]
Approved pending comment.

File: config/locales/jobseekers.yml (hunk 1 above)
Comment (alexbowen) [reply config/locales/jobseekers.yml:132]: i didnt chnage this but it is closed?

**Pull request merged**